builder/package.debian-11

```diff
@@ -36,6 +36,7 @@ depends:
 - libc6
 - libcairo2
 - libcurl4-openssl-dev
+- libdeflate0
```

Comment:
For the test failure, looks like `-ldeflate` ends up in the libR linker flags, which we test to make sure that packages like rJava can be built. Switching this to the dev package should get it working.